studio/pages/account/security.tsx

```diff
@@ -0,0 +1,246 @@
+import { AddNewFactorModal } from 'components/interfaces/Account'
+import { AccountLayout } from 'components/layouts'
+import AlertError from 'components/ui/AlertError'
+import ConfirmationModal from 'components/ui/ConfirmationModal'
+import { FormHeader } from 'components/ui/Forms'
+import ShimmeringLoader from 'components/ui/ShimmeringLoader'
+import { useMfaListFactorsQuery } from 'data/profile/mfa-list-factors-query'
+import { useMfaUnenrollMutation } from 'data/profile/mfa-unenroll-mutation'
+import { useStore } from 'hooks'
+import { useState } from 'react'
+import { NextPageWithLayout } from 'types'
+import {
+  Accordion,
+  AlertDescription_Shadcn_,
+  AlertTitle_Shadcn_,
+  Alert_Shadcn_,
+  Badge,
+  Button,
+  IconAlertCircle,
+  IconAlertTriangle,
+  IconSmartphone,
+  IconTrash,
+  Modal,
+} from 'ui'
+
+const Security: NextPageWithLayout = () => {
+  const { data } = useMfaListFactorsQuery()
+
+  return (
+    <div className="1xl:px-28 mx-auto flex flex-col px-5 pt-6 pb-14 lg:px-16 xl:px-24 2xl:px-32">
+      <div className="flex items-center justify-between">
+        <FormHeader
+          title="Multi-factor authentication"
+          description="Add an additional layer of security to your account by requiring more than just a password to sign in."
+        />
+      </div>
+      <Accordion
+        openBehaviour="multiple"
+        chevronAlign="left"
+        type="bordered"
+        className="max-w-4xl"
+        defaultValue={['auth-app']}
+      >
+        <Accordion.Item
+          header={
+            <div className="flex flex-row justify-between items-center p-2 w-full">
+              <div className="flex flex-row gap-2 items-center p-2">
+                <IconSmartphone />
+                <span className="text-lg">Authenticator app</span>
+              </div>
+
+              {data ? (
+                <>
+                  {data.totp.length === 0 && <Badge color="red">0 apps configured</Badge>}
+                  {data.totp.length === 1 && <Badge color="yellow">1 apps configured</Badge>}
+                  {data.totp.length === 2 && <Badge color="green">2 apps configured</Badge>}
+                </>
+              ) : null}
+            </div>
+          }
+          id="auth-app"
+          className="flex flex-row gap-8"
+        >
+          <ProfileCard />
+        </Accordion.Item>
+      </Accordion>
+    </div>
+  )
+}
+
+Security.getLayout = (page) => (
+  <AccountLayout title="Security" breadcrumbs={[{ key: 'security', label: 'Security' }]}>
+    {page}
+  </AccountLayout>
+)
+
+export default Security
+
+const ProfileCard = () => {
```

Comment:
could we shift all these components into `components/interfaces/Account/Security`?

Reply:
also nit: ProfileCard sounds odd as the component name here haha
maybe TOTPFactors?